org.eclipse.m2e.sdk.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="org.eclipse.m2e.sdk.feature"
       label="%featureName"
-      version="2.7.0.qualifier"
+      version="2.7.100.qualifier"
```

Comment:
The feature version has already been bumped this release cycle.
```suggestion
      version="2.7.0.qualifier"
```

Reply:
I will revert the version bump and force qualifier update as the first